cluster/config-defaults.yaml

```diff
@@ -350,7 +350,9 @@ teapot_admission_controller_namespace_delete_protection_enabled: "true"
 {{else if eq .Cluster.Environment "e2e"}}
 teapot_admission_controller_validate_application_label: "false"
 teapot_admission_controller_validate_base_images: "false"
-teapot_admission_controller_validate_pod_images: "false"
+teapot_admission_controller_validate_pod_images: "true"
+teapot_admission_controller_validate_pod_images_namespaces: "^(image-policy-test-enabled-.*|kube-system|visibility)$"
```

Comment:
This seems to be unused, is that a typo?

Reply:
Might be a good idea to rename the field as well, to prevent people from configuring it accidentally. Alternatively, we could also just ignore it completely in the template for production clusters.

Reply:
Discussed internally: Those settings are limited to e2e clusters. Usability and safety improvements are addressed in https://github.com/zalando-incubator/kubernetes-on-aws/pull/4582.